docker-compose.yml

```diff
@@ -105,3 +105,15 @@ services:
       - ./netkan:/home/netkan/netkan
       - ${HOME}/ckan_cache:/home/netkan/ckan_cache
     command: clean-cache --days 1
+  counts:
+    build:
+      context: netkan/.
+      target: dev
+    environment:
+      SSH_KEY: ${CKAN_NETKAN_SSHKEY}
+      NETKAN_REPO: ${CKAN_METADATA_PATH}
+      CKANMETA_REPO: ${CKAN_METADATA_PATH}
```

Comment:
Shouldn't `NETKAN_REPO` and `CKANMETA_REPO` have different values?

Reply:
It's only for Dev and probably. But they can be the same, my testing repo has a single NetKAN in it. I messed up the commit logic after waiting for the update to finish successfully and didn't want to wait 😂 I'll fix it before I merge 🙂